src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -53,12 +54,12 @@ public Line findById(Long id) {
     }
 
     public void update(Line newLine) {
-        String sql = "update LINE set name = ?, color = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getId()});
+        String sql = "update LINE set name = ?, color = ?, extra_fare = ? where id = ?";
+        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getExtraFare(), newLine.getId()});
```

Comment:
```suggestion
        jdbcTemplate.update(sql, newLine.getName(), newLine.getColor(), newLine.getExtraFare(), newLine.getId());
```

이미 아시는 내용일테지만, Object 배열 없이 가변인자를 활용할 수 있답니다 :)

Reply:
오호..! 사실 저 코드는 제가 작성한 코드는 아니어서요 😂 그래서 사실 `new Object[]{ }`로 작성할 생각을 못해봤어요...!
하나 궁금한 점이 그렇다면 저 `new Object[]`가 사용되는 경우가 언제 있는 걸까용?!
차이점을 찾아보려고 검색해봐도 잘 모르겠습니당 ㅠㅠ

Reply:
불필요한 `NEW Object[]`는 삭제하도록 수정하겠습니다 🙌